docs/persistence/bigquery.md

```diff
@@ -0,0 +1,53 @@
+# Bigquery Datasets
+
+!!! info
+    This feature is only available in GCP clusters.
+
+## NAIS Application yaml manifest options
+Full documentation of all available options can be found over at: [`spec.gcp.bigQueryDatasets[]`](../../nais-application/application#gcpbigquerydatasets).
+
+Example of an application using a `nais.yaml` provisioned BigQuery Dataset can be found here: [testapp](https://github.com/nais/testapp/blob/master/pkg/bigquery/bigquery.go).
+
+## Minimal Working Example
+=== "You can request a MWE Google Cloud BiqQuery Dataset through the NAIS Application manifest like so:"
+    ```yaml
+    apiVersion: "nais.io/v1alpha1"
+    kind: "Application"
+    metadata:
+      name: app-a
+    ...
+    spec:
+      ...
+      gcp:
+        bigQueryDatasets:
+          - name: my_bigquery_dataset
+            permission: READWRITE
+    ```
+
+## Caveats to be aware of
+
+=== "Automatic Deletion"
+    Once a BigQuery Dataset is provisioned, it will not be automatically deleted - unless one explicitly sets [`spec.gcp.bigQueryDatasets[].cascadingDelete`](../../nais-application/application#gcpbigquerydatasetscascadingdelete) to `true`.
+    This means that any cleanup must be done manually.  
+    <br/>
+    When there exist no tables in the specified BigQuery Dataset, deleting the "nais application" will delete the whole BigQuery Dataset, even if [`spec.gcp.bigQueryDatasets[].cascadingDelete`](../../nais-application/application#gcpbigquerydatasetscascadingdelete) is set to `false`.
+=== "Unique names"
+    The name of your Dataset must be unique within your team's GCP project.
+=== "Updates/Immutability"
+    The NAIS Manifest does not currently support updating any setting of existing BigQuery Datasets.  
+    <br/>
+    Thus, if you want a read-connection to a already-created BigQuery Dataset, take a look at [nais/dp](https://github.com/nais/dp/#dp).
```

Comment:
Tror det er litt prematurt å lenke til dp-repoet foreløpig. Hadde vel uansett vært best å lenke til nettsiden og ikke kode-repoet?

Reply:
Kjenner ikke til nettsiden - tenker at lenken til repoet med advarselen om prematur i Readme passer bra for nå.

Kan godt byttes ut med annen dp referanse, men ikke fjernes synes jeg.